ak-profile.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Akash0816",
+        "email": "akashoffical321@gmail.com"
+    },
+    "record": {
+        "CNAME": "Akash0816.github.io"
```

`CNAME` record must be lowercased
```suggestion
        "CNAME": "akash0816.github.io"
```

---

Doesn't have to be, but for beautification purposes, yes.